packages/console/src/components/CreateTenantModal/index.tsx

```diff
@@ -49,9 +49,9 @@ function CreateTenantModal({ isOpen, onClose }: Props) {
 
   const cloudApi = useCloudApi();
 
-  const createTenant = async (data: CreateTenantData) => {
-    const { name, tag } = data;
-    const newTenant = await cloudApi.post('/api/tenants', { body: { name, tag } });
+  const createTenant = async ({ name, tag, regionName }: CreateTenantData) => {
+    // @ts-expect-error will be removed once we bump the `@logto/cloud` version
```

Comment:
nit: better add a TODO tag incase we forget it.

Reply:
TODO here won't trigger the warning, and once we bump the version it will report an error. but agreed, better to add another line of comment including `// TODO:`. i'll skip this time since it has been removed in #5929